### DISCUSSION TITLE
DNS Entry of a partecipant - resolution inside SMP server

### DISCUSSION THREAD

**Discussion by francescodiperna** [phax/phoss-smp#220] [General]
Good morning, again I need you help to undersand a critical point.
For the partecipant 0211:it01928980679 the peppol network create a DNS url like this: 
http://B-7c4205b8f42e26c6cbecb20a34a11213.iso6523-actorid-upis.acc.edelivery.tech.ec.europa.eu
This URL point, at moment, to my root SMP test server.

Do I need to configure my SMP server to convert the url in the following:
http://B-7c4205b8f42e26c6cbecb20a34a11213.iso6523-actorid-upis.acc.edelivery.tech.ec.europa.eu/iso6523-actorid-upis::0211:it01928980679 ?
Is it right?

Kind regards

**Comment by phax** [comment]
Hi Francesco,
no, this is not your job.
Its the job of the SMP client (sending AP), to use the participant ID both for the DNS name calculation AND appending it to the path.

The URL `http://b-7c4205b8f42e26c6cbecb20a34a11213.iso6523-actorid-upis.acc.edelivery.tech.ec.europa.eu/` itself is useless - it just  points to the root of your SMP.

hth, Philip

**Reply by phax** [reply]
And don't use a forward to `https` in your SMP...

**Reply by francescodiperna** [reply]
Thanks
